test.js

```diff
@@ -0,0 +1,86 @@
+const ethers = require('ethers');
+const crypto = require('crypto');
+const axios = require('axios');
+
+//var id = crypto.randomBytes(32).toString('hex');
+//var privateKey = "0x"+id;
+//console.log("SAVE BUT DO NOT SHARE THIS:", privateKey);
+
+async function test() {
+	let quote, nonce, message, signature;
+
+	const wallet = new ethers.Wallet(process.env.TEST_PRIVATE_KEY);
+	console.log("Address: " + wallet.address);
+
+	// getQuote
+	await axios.post(`http://localhost:8081/getQuote`, {
+		type: "arweave",
+		userAddress: wallet.address,
+		files: [{length: 1256}, {length: 5969}],
+		payment: {
+			chainId: 80001,
+			tokenAddress: "0x0000000000000000000000000000000000001010",
+		},
+	})
+	.then((response) => {
+		quote = response.data;
+		console.log(`Got quote with id: ${quote.quoteId}`);
+	})
+	.catch((error) => {
+		console.error(error);
+		process.exit(1);
+	});
+
+	// upload
+	nonce = Math.floor(new Date().getTime()) / 1000;
```

Comment:
Why divide by 1000 here?

Reply:
`getTime()` gives the unix time in milliseconds. Divide by 1000 so we get fractional seconds in a float.